examples/basic/src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from 'react-dom';
+import WordCloud from 'react-d3-cloud';
+
+const data = [
+  { text: 'Hey', value: 1000 },
+  { text: 'lol', value: 200 },
+  { text: 'first impression', value: 800 },
+  { text: 'very cool', value: 1000000 },
+  { text: 'duck', value: 10 },
```

Comment:
It might be better to make the value range lies between 0 ~ 100, or change the mapper to a less-sensitive-to-large-number function, like `log`